Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Puree",
+    platforms: [
+        .iOS(.v10),
```

Comment:
We needn't support such an ancient version 🦖 
Our support policy is not fixed. but it's OK only above .v12.